_layouts/address.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+{% include head.html %}
+
+<body>
+  {% include header-extended.html %}
+
+  <main id="main-content">
+    <div class="grid-container flex-align-center margin-y-5">
+      <div class="display-flex flex-column flex-align-center">
+        <form class="usa-form usa-form--large">
+          <fieldset class="usa-fieldset">
+            <legend class="usa-legend">
+              <h1 class="margin-y-1">
+                {{ page.title }}
+              </h1>
+            </legend>
+            <p class="margin-y-1">
+              Required fields are marked with an asterisk (<abbr title="required" class="usa-hint usa-hint--required">*</abbr>).
+            </p>
+            <h2 class="margin-top-6">Contact Details</h2>
+            {% for label in page.contact-labels %}
+              <label class="usa-label" for="{{label.input-name}}">
```

Comment:
It's nice to keep spacing with template placeholders consistent. Our convention so far has been to include a space before and after for variables like `{{ page.title }}`.